docs/en/faq/how-kotlin-jdsl-is-different-from-jooq-and-querydsl.md

```diff
@@ -0,0 +1,35 @@
+# How is Kotlin JDSL different from jOOQ and QueryDSL?
+
+Unlike jOOq and QueryDSL, Kotlin JDSL does not generate a `Metadata Model (code generation) for writing queries.
+
+If you use code generation, you'll have to do the following:
+
+```
+Modify tables and entities →
+Entity modification causes compile errors →
+Run Manven or Gradle task to regenerate the metadate →
```

Comment:
```suggestion
Run Maven or Gradle task to regenerate the metadata →
```
Could you please fix it in Korean documentation as well?

Reply:
Well, I was in a hurry and missed some parts haha... I'll correct it.